src/pet.js

```diff
@@ -0,0 +1,73 @@
+const MINIMUM_HUNGER = 0;
+const MAXIMUM_FITNESS = 10;
+const INITIAL_AGE = 0;
+const INITIAL_HUNGER = 0;
+const INITIAL_FITNESS = 10;
+const GROWUP_AGE = 1;
+const GROWUP_HUNGER = 5;
+const GROWUP_FITNESS = 3;
+const WALK_FITNESS = 4;
+const FEED_HUNGER = 3;
+function Pet(name) {
+    this.name = name;
+    this.age = INITIAL_AGE;
+    this.hunger = INITIAL_HUNGER;
+    this.fitness = INITIAL_FITNESS;
+//    this.children = [];
```

Comment:
No need to comment this out, it was working!
Also, a value is is assigned for `checkupResult` on this object so that should be declared here too